_config.yml

```diff
@@ -1,13 +1,13 @@
 # Site settings
 title: Sensu Plugins
 
-support_email: support@sensuapp.org
-irc: irc.freenode#sensu
+support_email: community@sensu.io
+slack: slack.sensu-plugins.io
```

Comment:
should it be slack.sensu.io ? Just trying to be consistent. I see that one used below.

Reply:
🤷‍♂️  they both work.

Reply:
Fixed.